daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md

```diff
@@ -42,7 +42,7 @@ The following example shows how to save two key/value pairs in a single call usi
 {{% codetab %}}
 Begin by ensuring a Dapr sidecar is running:
 ```bash
-dapr --app-id myapp --port 3500 run
+dapr run --app-id myapp --dapr-http-port 3500 run
```

Comment:
```suggestion
dapr run --app-id myapp --port 3500
```

The trailing run is not necessary. The previous line says `Begin by ensuring a Dapr sidecar is running:`, so no app is running only the dapr sidecar.